app/javascript/components/patient/Patient.js

```diff
@@ -68,665 +111,614 @@ class Patient extends React.Component {
-          <Card.Body className="mx-0 px-0 my-0 py-0">
-            <Row className="g-border-bottom-2 mx-2 pb-4 mb-2"></Row>
-            <Row className="g-border-bottom-2 pb-4 mb-2 mt-4 mx-1">
-              <Col id="address" md="7">
-                <Row>
-                  <Col>
-                    <div className="float-left">
-                      <div className="h4">
-                        <b>ADDRESS</b>
-                      </div>
-                    </div>
-                    <div>
-                      {this.props.goto && (
-                        <Button variant="link" className="pt-0" onClick={() => this.props.goto(1)} aria-label="Edit Button">
-                          <div className="h5">(Edit)</div>
-                        </Button>
-                      )}
-                    </div>
-                    <div className="clearfix"></div>
-                  </Col>
-                </Row>
-                <Row>
-                  <Col className="text-truncate">
-                    {!showDomesticAddress && !showForeignAddress && (
-                      <Row className="py-1">
-                        <Col className="text-truncate">
-                          <span className="none-text">None</span>
-                        </Col>
-                      </Row>
-                    )}
-                    {(showDomesticAddress || showForeignAddress) && (
-                      <Row className="py-1">
-                        <Col>
-                          <b>HOME ADDRESS</b>
-                        </Col>
-                      </Row>
+        {!this.props.editMode && (
+          <div className="details-expander">
+            <Button
+              id="details-expander-link"
+              variant="link"
+              className="p-0"
+              aria-label="Edit Notes"
+              onClick={() => this.setState({ expanded: !this.state.expanded })}>
+              <FontAwesomeIcon className={this.state.expanded ? 'chevron-opened' : 'chevron-closed'} icon={faChevronRight} />
+              <span className="pl-2">{this.state.expanded ? 'Hide' : 'Show'} address, travel, exposure, and case information</span>
+            </Button>
+            <span className="dashed-line"></span>
+          </div>
+        )}
+        <Collapse in={this.state.expanded}>
+          <div>
+            <Row>
+              <Col id="address" lg={14} xl={12} className="col-xxxl-10">
+                <div className="section-header">
+                  <h4 className="section-title">Address</h4>
+                  <div className="edit-link">
+                    {this.props.goto && (
+                      <Button variant="link" id="edit-address-btn" className="py-0" onClick={() => this.props.goto(1)} aria-label="Edit Address">
+                        Edit
+                      </Button>
                     )}
+                  </div>
+                </div>
+                {(showDomesticAddress || showMonitoredAddress) && (
```

Comment:
Previously if neither of these options were true we would show "None", it doesn't look like that is still happening here (though you still are elsewhere). Same with the foreign address below.

Reply:
Based on Mike's mockups and my own investigating, I thought at least one of the four address fields were required so we would never encounter this scenario.  I added a check in my latest commit just in case.